Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM datastax/dse-server:6.0.0
+FROM datastax/dse-server:6.0.1
```

Comment:
why not 6.0.2?

Reply:
Only 6.0.1 was available at the time I made the change.  If 6.0.2 is available I can change it easily enough, but not tested yet.

Reply:
Looks like 6.0.2 is not yet available